pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1246,6 +1246,8 @@ func GetPodAntiAffinityTerms(podAntiAffinity *v1.PodAntiAffinity) (terms []v1.Po
 	return terms
 }
 
+// getMatchingTopologyPairs is an initialization helper to calculate on each existing pod
```

Comment:
This comment is exactly same as with `getMatchingTopologyPairsOfExistingPod`, copy-paste error? :D

Reply:
Not _exactly_ :) The other one is for **all** nodes, this one if for **a single given** node.

Reply:
Oh, I must be blind.